### PR TITLE
Fix up a load of urls that have www, http, or /certification

### LINKED ISSUE #9855: Broken link / redirect
## Summary

This page links to a page that doesn't exist: https://ubuntu.com/legal/ubuntu-advantage-service-description

## Process

Go to the page referenced above, find the link to http://www.ubuntu.com/certification/ and click it.

## Current and expected result

Current result is 404 error: https://ubuntu.com/certification?_ga=2.17078905.1798568884.1622549587-1520193984.1566474324

Expected result is a redirect to https://certification.ubuntu.com/

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Fix up a load of urls that have www, http, or /certification
- Removed the google analytics tracker from urls `?_ga(.*>)`
- Fixed up the UASD copy doc

## QA

- Check out this feature branch
- Read through the code and confirm changes look good

## Issue / Card

Fixes #9855

### CODE REVIEW

File: templates/desktop/partners.html [hunk 1]
```diff
@@ -114,7 +114,7 @@ <h3>Ongoing support</h3>
     <div class="col-6">
       <h2>Ubuntu Desktop certified hardware</h2>
       <p>To offer top performance on Ubuntu, manufacturers are pre-loading devices with an Ubuntu Certified image. This ensures driver integration and superior device performance.  Canonical is committed to working closely with with the world's leading manufacturers to optimise and certify Ubuntu on their laptops and PCs.</p>
-      <p><a class="p-link--external" href="https://certification.ubuntu.com/certification/">Learn more about Ubuntu Certified hardware</a></p>
+      <p><a href="/certified">Learn more about Ubuntu Certified hardware</a></p>
```
Comment (anthonydillon) [inline templates/desktop/partners.html:117]: Should this now have `&nbsp;&rsaquo;`

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: one small comment inline

File: templates/desktop/partners.html (hunk 1 above)
Comment (pmahnke) [reply templates/desktop/partners.html]: good catch- updated

**Comment by anthonydillon** [comment]
Good to go :+1:

**Comment by anthonydillon** [comment]
@pmahnke are you planning to revisit this PR?

**Comment by pmahnke** [comment]
@anthonydillon - @nottrobin is looking into why the python test is failing

**Pull request merged**